### DISCUSSION TITLE
Organizing and filtering A/V content

### DISCUSSION THREAD

**Discussion by johnmcd88** [osmandapp/OsmAnd#14306] [Q&A] [answered]
[Note: This was cross-posted on the OpenStreetMap forums - I wasn't sure which one I should use.]

Hi all,

I've been an avid user of OSMAND+ on Android for quite a while, and I've used it extensively to plan, record and document various hikes, backpacking trips, bike trips, vacations, etc. I tend to do many of my trips in a couple of areas, and one thing I've noticed is that my maps are starting to get kind of crowded with pictures, videos and voice recordings in those areas. Is there some way to filter what gets displayed using something like a keyword (e.g. 'Aug-2021-WM-Hike'), grouping, date range, layer, etc. to only show the icons for A/V content related to a specific track or date range? I've spent a lot of time digging through the various docs and options and I couldn't find anything, but there are parts of OSMAND+ that I still don't fully understand.

Many thanks to everyone that's contributed to this fantastic tool and its supporting ecosystem!

John

**Comment by vshcherb** [comment, marked as answer]
Unfortunately there is no such option yet and nothing comes to my mind as a workaround to this.

**Comment by vshcherb** [comment]
We have further plans to combine audio / video notes with Favorites, so it will make them more customizable.

**Reply by johnmcd88** [reply]
Thank you.
